### DISCUSSION TITLE
Concerns regarding the trademark proposal from the Rust Foundation

### DISCUSSION THREAD

**Discussion by Desdaemon** [fzyzcjy/flutter_rust_bridge#1167] [General]
Recently the Rust Foundation made a [proposal](https://www.reddit.com/r/rust/comments/12e7tdb/rust_trademark_policy_feedback_form/), which among other things would prohibit usage of the word "rust" anywhere in a crate name, citing desire to assert ownership over such crates. Regardless of whether this proposal is accepted, which is unlikely, it limits the possibility for this library to continue using the word "rust". Should we be concerned about this, or do you think it's not going to affect flutter_rust_bridge going forward?

**Comment by fzyzcjy** [comment]
Thanks for pointing out the information! Hmm if the proposal is accepted then seems we need to follow it. On the other hand, IMHO currently we can still use this name, since it is quite intuitive to everyone who wants to have a bridge between Flutter and Rust ;)
